pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -203,6 +203,33 @@ func (ds *dockerService) ListPodSandbox(filter *runtimeApi.PodSandboxFilter) ([]
 	return result, nil
 }
 
+// applyLinuxSpecificOptions applies LinuxPodSandboxConfig to dockercontainer.HostConfig and dockercontainer.ContainerCreateConfig.
+func (ds *dockerService) applyLinuxSpecificOptions(hc *dockercontainer.HostConfig, lc *runtimeApi.LinuxPodSandboxConfig, createConfig *dockertypes.ContainerCreateConfig, image string) error {
+	// Apply Cgroup options.
+	// TODO: Check if this works with per-pod cgroups.
+	hc.CgroupParent = lc.GetCgroupParent()
+
+	// Verify RunAsNonRoot of security context.
```

Comment:
For simplicity

- The verification is related with docker client ( it calls `ds.isImageRoot` to query image user), while applySandboxSecurityContext doesn't require the client.
- applySandboxSecurityContext never error out